ProcessLib/HeatTransportBHE/BHE/CreateBHECoaxial.cpp

```diff
@@ -34,11 +34,11 @@ parseBHECoaxialConfig(
 {
     // if the BHE is using python boundary condition
     auto const bhe_if_use_python_bc_conf =
+        //! \ogs_file_param{prj__processes__process__HEAT_TRANSPORT_BHE__borehole_heat_exchangers__borehole_heat_exchanger__bhe_if_use_python_bc}
         config.getConfigParameter<bool>("bhe_if_use_python_bc", false);
```

Comment:
The keyword itself could also be improved, as mentioned in the original PR.
Something like `enable_bhe_pipe_network`, or `use_bhe_pipe_network`.

Reply:
Yes, the keyword has been changed to 'use_bhe_pipe_network' after discussing with Shuang Chen.